controllers/image_generator.go

```diff
@@ -295,6 +301,11 @@ func (r *ImageGenerator) createJobFromFile(jobFileName string) (*batchv1.Job, er
 				Name:  "AMI_ID",
 				Value: imageId,
 			})
+		} else if r.provider == LibvirtProvider {
+			job.Spec.Template.Spec.Containers[0].Env = append(job.Spec.Template.Spec.Containers[0].Env, corev1.EnvVar{
+				Name:  "LIBVIRT_IMAGE_ID",
```

Comment:
s/LIBVIRT_IMAGE_ID/IMAGE_ID" based on previous comments w.r.to the var for delete job